cmd/recorder/recorder.go

```diff
@@ -181,6 +201,13 @@ func (rec *Recorder) runBrowser(recURL string) error {
 				return nil
 			}
 			continue
+		case <-rec.transcoderStartedCh:
+			if err := chromedp.Run(ctx,
+				chromedp.Evaluate("window.callsClient?.ws?.send('job_started');", nil),
```

Comment:
Slightly too clever for my taste but it was the quickest way to achieve what I needed. I am actually thinking of redesigning this a bit and make a proper HTTP endpoint since we have another requirement to inform back the plugin side in case of pre-connection failures. Will think some more on this.